documentation/deploy-page.md

```diff
@@ -0,0 +1,10 @@
+# Deploy page
+
+Right now the offline page is a sign that is enabled via toggle, and with mvc offline.
+To see how to enable and disable the deploy page check out this [example](https://www.screencast.com/t/FizkXB8kt0iJ)
+
+Current offline page is located [here](https://app.fromdoppler.com/offline)
+
+Configurations available in toggle file:
+- Offline: true (set the app in offline mode), false (set the app in online mode)
+- Languages: spanish only
```

Comment:
In general, is a good practice using an interline between a paragraph and a list.

```suggestion
Configurations available in toggle file:

- Offline: `true` (set the app in offline mode), `false` (set the app in online mode)
- Languages: Spanish only
```

Reply:
Here you have some advice about using Markdown in VS Code: https://code.visualstudio.com/docs/languages/markdown

I really like Markdown Lint extension, it is very didactical.

Reply:
I have it installed!